tensorboard/webapp/runs/effects/runs_effects.ts

```diff
@@ -89,84 +67,6 @@ export class RunsEffects {
     private readonly runsDataSource: RunsDataSource
   ) {}
 
-  private getRunsListLoadState(experimentId: string): Observable<LoadState> {
-    return this.store.select(getRunsLoadState, {experimentId}).pipe(take(1));
-  }
-
-  private getExperimentsWithLoadState(
-    experimentIds: string[],
-    loadStateMatcher: (loadState: DataLoadState) => boolean
-  ) {
-    return forkJoin(
-      experimentIds.map((eid) => {
-        return this.getRunsListLoadState(eid);
-      })
-    ).pipe(
-      map((loadStates) => {
-        return experimentIds.filter((unused, index) => {
-          return loadStateMatcher(loadStates[index].state);
-        });
-      })
-    );
-  }
-
-  private readonly experimentsWithStaleRunsOnRouteChange$ = this.actions$.pipe(
-    ofType(navigated),
-    withLatestFrom(this.store.select(getActiveRoute)),
-    distinctUntilChanged(([, prevRoute], [, currRoute]) => {
-      return areSameRouteKindAndExperiments(prevRoute, currRoute);
-    }),
-    withLatestFrom(this.store.select(getExperimentIdsFromRoute)),
-    filter(([, experimentIds]) => !!experimentIds),
-    map(([, experimentIds]) => experimentIds!),
-    mergeMap((experimentIds) => {
-      return this.getExperimentsWithLoadState(experimentIds, (state) => {
-        return (
-          state === DataLoadState.FAILED || state === DataLoadState.NOT_LOADED
-        );
-      }).pipe(
-        map((experimentIdsToBeFetched) => {
-          return {experimentIds, experimentIdsToBeFetched};
-        })
-      );
-    })
-  );
-
-  private readonly experimentsWithStaleRunsOnReload$ = this.actions$.pipe(
-    ofType(coreActions.reload, coreActions.manualReload),
-    withLatestFrom(this.store.select(getExperimentIdsFromRoute)),
-    filter(([, experimentIds]) => !!experimentIds),
-    map(([, experimentIds]) => experimentIds!),
-    mergeMap((experimentIds) => {
-      return this.getExperimentsWithLoadState(experimentIds, (state) => {
-        return state !== DataLoadState.LOADING;
-      }).pipe(
-        map((experimentIdsToBeFetched) => {
-          return {experimentIds, experimentIdsToBeFetched};
-        })
-      );
-    })
-  );
-
-  /**
-   * Fetches runs on navigation or in-app reload.
-   *
-   * @export
-   */
-  loadRunsOnNavigationOrReload$ = createEffect(
-    () => {
-      return merge(
-        this.experimentsWithStaleRunsOnRouteChange$,
-        this.experimentsWithStaleRunsOnReload$
```

Comment:
There are some other changes going on with this code base (at least #6522) so please make sure to incorporate those changes when you rebase/merge.